azure/backgroundJobTemplate.json

```diff
@@ -0,0 +1,177 @@
+    "settingsAuthenticationSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "Auth secret for manage back-end."
+      }
+    },
+    "settingsManageApiSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "The secret shared between manage courses backend and manage courses API to authorize requests."
+      }
+    },
+    "settingsSearchApiSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "The secret shared between manage courses backend and search and compare API to authorize requests."
+      }
+    }
+  },
+  "variables": {
+    "deploymentUrlBase": "https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/templates/"
+  },
+  "resources": [
+    {
+      "name": "container-instances",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2017-05-10",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('deploymentUrlBase'), 'container-instances.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "containerInstanceName": {
+            "value": "[parameters('containerInstanceName')]"
+          },
+          "containerName": {
+            "value": "[parameters('containerName')]"
+          },
+          "imageName": {
+            "value": "[parameters('imageName')]"
+          },
+          "environmentVariables": {
+            "value": [
+              {
+                "name": "RAILS_ENV",
+                "value": "[parameters('railsEnv')]"
+              },
+              {
+                "name": "AUTHENTICATION_TOKEN",
+                "secureValue": "[parameters('authenticationToken')]"
+              },
+              {
+                "name": "DB_DATABASE",
+                "value": "[parameters('databaseName')]"
+              },
+              {
+                "name": "DB_HOSTNAME",
+                "value": "[parameters('databaseServerHostName')]"
+              },
+              {
+                "name": "DB_PASSWORD",
+                "secureValue": "[parameters('databasePassword')]"
+              },
+              {
+                "name": "DB_USERNAME",
+                "value": "[parameters('databaseUsername')]"
+              },
+              {
+                "name": "DB_PORT",
+                "value": "[parameters('databasePort')]"
+              },
+              {
+                "name": "SECRET_KEY_BASE",
+                "secureValue": "[parameters('secretKeyBase')]"
+              },
+              {
+                "name": "SENTRY_DSN",
+                "secureValue": "[parameters('sentryDSN')]"
+              },
+              {
+                "name": "SETTINGS__AUTHENTICATION__SECRET",
+                "secureValue": "[parameters('settingsAuthenticationSecret')]"
+              },
+              {
+                "name": "SETTINGS__MANAGE_API__SECRET",
```

Comment:
This is used when communicating with `manage-courses-api`. I believe the only outgoing connection we'll make in this app will be to search-and-compare so this is one we can probably get rid of.